Representation.php

```diff
@@ -88,13 +88,6 @@ function construct_flags($period, $adaptation_set, $representation){
         $codecs = 0;
     $processArguments .= ' -codecs ' . $codecs;
 
-    if ((substr($codecs, 0, 4) == "ac-3") or
-        (substr($codecs, 0, 4) == "ec-3") or
-        (substr($codecs, 0, 4) == "ac-4"))
-    {
-        $processArguments .= ' -dolby';
-    }
-
```

Comment:
It seems that this fix will only disable the tests under condition "-dolby" which would result in removing tests. Am I missing something?

Reply:
Please see the pull request https://github.com/Dash-Industry-Forum/ISOSegmentValidator/pull/41 
ISOSegmentValidator changed to use codecs flag instead of dolby so this flag is no longer required.

Reply:
Ah, I missed that. Makes sense now!